build/theater.js

```diff
@@ -223,10 +223,9 @@
 
         if (typeof scene === "string") {
           var params   = scene.split(":"),
-              hasActor = params.length > 1,
+              hasActor = (params.length > 1 & params[0][params[0].length-1] !== '#'),
               actor    = hasActor ? params[0].trim() : null,
-              speech   = hasActor ? params[1] : params[0];
-
+              speech   = hasActor ? (params.shift(), params.join(":")) : (params[0] = params[0].replace(/#+$/, ""), params.join(":"));
```

Comment:
It tends to get too complex here. Feel free to split it on several lines to make it more readable.

Also, `.replace(/#+$/, "")` will remove all the `#` in `I am your##: father` while it should remove only the preceding `#`. `.replace(/#$/, "")` would do the job. That been said, I'd suggest to join before replacing what's needed (for simplicity sake).

```javascript
params[0] = params[0].join(":").replace(/#:/g, ":");
```